tests/test_mig_shared_install.py

```diff
@@ -231,6 +231,26 @@ def test_creates_output_files_with_datasafety(self):
         self.assertConfigKey(
             actual_file, 'SITE', 'datasafety_text', expected='TEST_DATASAFETY_TEXT')
 
+    def test_creates_output_files_with_premanent_freeze(self):
```

Comment:
I'll fix the 'premanent' typo during merge.

Reply:
Thanks.